packages/components/src/templates/next/components/internal/Breadcrumb/Breadcrumb.tsx

```diff
@@ -5,23 +5,19 @@ const Breadcrumb = ({ links, LinkComponent = "a" }: BreadcrumbProps) => {
   return (
     <div className="flex flex-row flex-wrap gap-1">
       {links.map((link, index) => {
-        const isCurr = index === links.length - 1
+        const isLast = index === links.length - 1
         return (
           <div
             key={index}
-            className="flex flex-row items-center gap-1 text-content-medium"
+            className="text-content-medium flex flex-row items-center gap-1"
           >
-            {isCurr ? (
-              <p className="font-semibold text-content">{link.title}</p>
-            ) : (
-              <LinkComponent
-                href={link.url}
-                className="underline underline-offset-2 hover:text-hyperlink-hover active:text-hyperlink"
-              >
-                {link.title}
-              </LinkComponent>
-            )}
-            {!isCurr && <MdChevronRight className="h-auto min-w-6" />}
+            <LinkComponent
+              href={link.url}
+              className="hover:text-hyperlink-hover active:text-hyperlink underline underline-offset-2"
+            >
+              {link.title}
+            </LinkComponent>
```

Comment:
i think this might have some differences in styling? previously, the last element has `font-semibold text-content` but this is omitted now and it's styled similarly to every other element in our new rendition. 

both seem fine to me, i'll let you make the call?

Reply:
yup! now the last element is the current page's parent, so it should be a link and shouldn't have the semibold formatting